docs/guide/http_cache.md

```diff
@@ -757,3 +757,72 @@ fos_http_cache:
 #### Credits
 
 This feature is based on [Context aware HTTP caching post](http://asm89.github.io/2012/09/26/context-aware-http-caching.html) by [asm89](https://github.com/asm89).
```

Comment:
Can be updated to something like this as our approach has changed a bit away from the concept there.

` This feature was originally inspired by on [Context aware HTTP caching blog post](http://asm89.github.io/2012/09/26/context-aware-http-caching.html) by [asm89](https://github.com/asm89) and has since evolved in FOSHttpCache and eZ to Varnish restart instead of curl. But the overall approach is the same, and its a good read for better understanding it.`